lib/shopify_cli/theme/extension/dev_server.rb

```diff
@@ -71,15 +111,26 @@ def param_builder
 
         def setup_server
           CLI::UI::Frame.open(frame_title, color: :magenta, timing: nil) do
-            # TODO: https://github.com/Shopify/shopify-cli/issues/2538
-            ctx.open_url!(address)
             ctx.puts(preview_message)
           end
 
           watcher.start
           syncer.start
         end
 
+        def clean_last_line
+          # move the cursor to the start of the line
+          command = "\r"
```

Comment:
do you think it's worth adding a `start_of_line` command to the ANSI module?

Reply:
That's a very good idea! But, we've ended up removing this logic :(